extensions/kafka-streams/deployment/src/main/java/io/quarkus/kafka/streams/deployment/KafkaStreamsProcessor.java

```diff
@@ -196,4 +202,34 @@ void configureAndLoadRocksDb(KafkaStreamsRecorder recorder, KafkaStreamsRuntimeC
     AdditionalBeanBuildItem registerBean() {
         return AdditionalBeanBuildItem.unremovableOf(KafkaStreamsTopologyManager.class);
     }
+
+    @BuildStep
+    void addHealthChecks(KafkaStreamsBuildTimeConfig buildTimeConfig) {
+        healthChecks.produce(
+                new HealthBuildItem(
+                        "io.quarkus.kafka.streams.runtime.health.KafkaStreamsTopicsHealthCheck",
+                        buildTimeConfig.healthEnabled, "kafka-streams"));
+        healthChecks.produce(
+                new HealthBuildItem(
+                        "io.quarkus.kafka.streams.runtime.health.KafkaStreamsStateHealthCheck",
+                        buildTimeConfig.healthEnabled, "kafka-streams"));
+    }
+
+    private boolean isContainerBuild() {
```

Comment:
Where is this used?

Reply:
I don't know.
And I don't remamber to write this peace of code.
Maybe a bad previous confilct resolving.

I removed this method then run again integration tests and everything is working fine. So I think it's useless.

You have the :eye: of :tiger: :boxing_glove:!!

Reply:
Yes, I can confirm it's a left-over. Got obsoleted by 9a099abadd903219b2baa98612f1c740b7edfab8.